docs/faq.md

```diff
@@ -0,0 +1,148 @@
+
+A dedicated Virtual Cloud Network is provided for the meshStack. This network is completely encapsulated from other meshStack and their dedicated customer networks. The network configuration also includes the usage of Security Groups to strictly configure communication in the network.
+
+### Does meshStack support Single sign-on (SSO) and Multi-Factor-Authentication (MFA)?
+
+Yes, meshStack supports SSO and MFA.
+
+Activating SSO requires the integration between your own Enterprise Identity Provider (IDP) (e.g. Azure AD) with your respective meshStack.
+
+MFA must be activated in the Enterprise IDP which is integrated with your meshStack.
+
+### Are we allowed to security test meshStack on our own?
+
+Yes, you can perform security tests like Penetration Tests on meshStack but this always requires prior information and getting signed approval from meshcloud GmbH.
+
+### Can meshcloud GmbH employees (aka meshis) access our organizational data?
+
+No, meshis (employee at meshcloud) do not primarily have any access to your organizational data. Though, in certain scenarios such as for support cases if you would like to give the access or permission to your cloud environment then you can assign it to any of our relevant meshi's.
+
+## Data Protection
+
+### Does meshStack process any Personal Identifiable Information (PII)?
+
+meshStack stores your *First Name*, *Last Name*, *Email Address*, and *Last Login Date*. All these are stored in our database for user management (which is also managed via “a locally managed instance of ***[Keycloak](https://www.keycloak.org/)”***).
+
+The External User Identifier (like the Email Address) is replicated to the respective integrated cloud platforms, e.g. AWS, GCP, Azure, etc.
+
+### Is meshStack GDPR-compliant?
+
+Yes, meshStack is compliant with the official terms of the General Data Protection Regulation (GDPR). The official definitions are explained in the GDPR requirements as follows:
+
+- Art. 24 - [Responsibility of the controller](https://gdpr-info.eu/art-24-gdpr/)
+- Art. 25 - [Data protection by design and by default](https://gdpr-info.eu/art-25-gdpr/)
+- Art. 26 - [Joint controllers](https://gdpr-info.eu/art-26-gdpr/)
+- Art. 27 - [Representatives of controllers or processors not established in the Union](https://gdpr-info.eu/art-27-gdpr/)
+- Art. 28 - [Processor](https://gdpr-info.eu/art-28-gdpr/)
+
+### Does meshStack provide a compliant way to delete my PII and organizational data?
+
+Yes, meshStack deletes your data at your request, if the data is not collected by any legal archiving obligation. If the archiving obligation applies, we archive your data.
+
+### Where will my data be located?
+
+a. **Data used and generated by meshStack**
+
+meshStack uses different cloud providers such as AWS, GCP, Azure, etc. in order to store and backup data. These data can be located in the data centers of the respective cloud providers mainly in the EU region as well as across the globe.
```

Comment:
EU region. The geolocation can be configured differently on the customers request